ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml

```diff
@@ -2096,9 +2096,9 @@ objects:
             --parameters privateDNSZoneName="${CLUSTER_NAME}.${base_domain}" \
             --parameters baseName="$INFRA_ID"
 
-          MASTER0_IP=$(az network nic ip-config show -g $RESOURCE_GROUP --nic-name ${INFRA_ID}-master-0-nic --name pipConfig --query "privateIpAddress" -o tsv)
-          MASTER1_IP=$(az network nic ip-config show -g $RESOURCE_GROUP --nic-name ${INFRA_ID}-master-1-nic --name pipConfig --query "privateIpAddress" -o tsv)
-          MASTER2_IP=$(az network nic ip-config show -g $RESOURCE_GROUP --nic-name ${INFRA_ID}-master-2-nic --name pipConfig --query "privateIpAddress" -o tsv)
+          MASTER0_IP=$(az vm show -g $RESOURCE_GROUP -n ${INFRA_ID}-master-0 --show-details --query "privateIps" -o tsv)
```

Comment:
Do the internal load balancer IP addresses get pulled in here as well?

Reply:
No. I don't think that there are separate IP addresses for the backend of the load balancers. I don't see anything of that nature in the results  of `az network nic ip-config show`, and there is only one private IP address in the results of `az vm show`.

```
$ az vm show -g ci-ln-bbtq6vt-002ac-s26c2-rg -d -n ci-ln-bbtq6vt-002ac-s26c2-master-0 --query "privateIps" -o tsv
10.0.0.5
```

Reply:
Looks good. The public IP address for the bootstrap node, which has a public ip on the nic, can be the public load balancer. But you're right, that doesn't seem to be the case here for the private ip/internal LBs.

LGTM but I will hold off until you decide https://github.com/openshift/release/pull/14432#issuecomment-747812492